src/server/handlers/catch-all-handler.tsx

```diff
@@ -145,7 +145,7 @@ export default async (req: Request, res: Response) => {
       errorPageData,
       showAdultConsentModal:
         !!site?.site_view.site.content_warning &&
-        !(site.my_user || req.cookies[adultConsentCookieKey]),
```

Comment:
Logged in users should not be shown the consent modal.

Reply:
I was going by [lemmynsfw's changes](https://github.com/LemmyNet/lemmy-ui/compare/main...lemmynsfw:lemmy-ui:main) for this one. I doubt instances that will need the consent form will shift to using it after the instance has been established long enough to have a userbase.

Reply:
I pry agree that logged-in users don't need to be shown the popup. If they're already part of the instance they know what its about.